21:42:07.459 TRACE [32296] [FFSCENE_PYTHON] ffscene_python_t::open begin
21:42:07.716 TRACE [32296] [FFSCENE] ffscene_t::open begin
21:42:07.716 INFO  [32296] [FFRPC] ffrpc_t::connect_to_broker begin...host_<tcp://127.0.0.1:10241>,node_id_[0]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_msg_impl cmd[5] begin
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data node_id[6] begin
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::route_logic_msg_t::in_t] -> msg id[7]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::scene_call_msg_t::in_t] -> msg id[8]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::session_enter_scene_t::in_t] -> msg id[9]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::session_offline_t::in_t] -> msg id[10]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::open end ok m_node_id[6]
21:42:07.717 TRACE [32296] [FFSCENE] ffscene_t::open end ok
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::session_verify_t::in_t] -> msg id[11]
21:42:07.717 TRACE [32296] [FFSCENE_PYTHON] ffscene_python_t::open end ok
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data name[gate@0] -> node id[1],bind_broker_id[0]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data name[scene@0] -> node id[6],bind_broker_id[0]
21:42:07.717 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data end ok
21:42:07.718 TRACE [32296] [FFRPC] ffrpc_t::handle_msg_impl cmd[5] call end ok
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_msg_impl cmd[5] begin
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data node_id[0] begin
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::route_logic_msg_t::in_t] -> msg id[7]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::scene_call_msg_t::in_t] -> msg id[8]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::session_enter_scene_t::in_t] -> msg id[9]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::session_offline_t::in_t] -> msg id[10]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data msg name[ff::session_verify_t::in_t] -> msg id[11]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data name[gate@0] -> node id[1],bind_broker_id[0]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data name[scene@0] -> node id[6],bind_broker_id[0]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data name[scene@1] -> node id[12],bind_broker_id[0]
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_sync_data end ok
21:42:07.720 TRACE [32296] [FFRPC] ffrpc_t::handle_msg_impl cmd[5] call end ok
21:42:09.912 TRACE [32296] [FFRPC] ffrpc_t::handle_msg_impl cmd[6] begin
21:42:09.912 TRACE [32296] [FFRPC] ffrpc_t::handle_broker_route_msg msg_id[11],callback_id[1] begin
21:42:09.912 TRACE [32296] [FFSCENE] ffscene_t::process_session_verify begin session_key size=6
21:42:09.917 ERROR [32296] [FFSCENE_PYTHON] ffscene_python_t::gen_verify_callback exception<'NoneType' object has no attribute 'query'
Traceback (most recent call last):

  File "./pylib/ffext.py", line 94, in ff_session_verify
    ret = g_session_verify_callback(session_key, online_time, ip, gate_name)

  File "./pyproject/player_mgr/player_mgr_handler.py", line 11, in real_session_verify
    session_id = dbservice.get_session_by_auth_key(szAuthKey)

  File "./pyproject/db/dbservice.py", line 33, in get_session_by_auth_key
    import id_manager.idmanager as idmgr

  File "./pyproject/id_manager/idmanager.py", line 60, in <module>
    GenPlayerID = _PlayerIDMgr.gen_id()

  File "./pyproject/id_manager/idmanager.py", line 38, in gen_id
    self.update_id()

  File "./pyproject/id_manager/idmanager.py", line 56, in update_id
    self.db.query("UPDATE `id_generator` SET `AUTO_INC_ID` = '%d' WHERE `TYPE` = '%d' AND `SERVER_ID` = '%d' AND `AUTO_INC_ID` < '%d'" % (now_val, self.type_id, self.server_id, now_val), cb)

AttributeError: 'NoneType' object has no attribute 'query'

>
21:42:09.917 TRACE [32296] [FFSCENE] ffscene_t::process_session_verify end ok
21:42:09.917 INFO  [32296] [FFRPC] ffrpc_t::handle_broker_route_msg end ok msg_.bridge_route_id=0
21:42:09.917 TRACE [32296] [FFRPC] ffrpc_t::handle_msg_impl cmd[6] call end ok
21:42:09.917 INFO  [32296] [FFRPC] ffbroker_t::send_to_broker_by_nodeid begin bridge_route_id_[0]
21:42:09.918 TRACE [32296] [FFRPC] ffrpc_t::send_to_broker_by_nodeid dest_node_id[1], broker_node_id[0], msgid<0>, callback_id_[1] end ok
